test/trento/application/projectors/host_projector_test.exs

```diff
@@ -166,6 +167,28 @@ defmodule Trento.HostProjectorTest do
                      1000
   end
 
+  test "should update the selected_checks field when event is received" do
+    %{id: host_id} = insert(:host, id: UUID.uuid4())
+
+    test_checks_data = [
```

Comment:
Not a big fan of this name. I would go with `cases` or `scenarios`.
Anyway, not strong opinions at all

Reply:
Fair enough :D